code/Sitecore.NSubstitute/SiteInfoPropertiesBuilder.cs

```diff
@@ -0,0 +1,186 @@
+﻿using Sitecore.Data;
+using StringDictionary = Sitecore.Collections.StringDictionary;
+
+namespace Sitecore.NSubstituteUtils
+{
+    public class SiteInfoPropertiesBuilder

```

Comment:
It is very convenient to use implicit casts for FakeItem -> Item.
Could we have a similar operator for SiteInfoPropertiesBuilder?

Reply:
Done. Implicit casts have been added